src/app/services/map-view.service.ts

```diff
@@ -20,10 +22,24 @@ export class MapViewService {
 
   // Define base colors
   COLORS = {
-    red: [255, 0, 0, 0.9],
-    green: [0, 255, 0, 0.9],
-    blue: [0, 0, 255, 0.9]
+    red: [255, 0, 0, 1],
+    green: [0, 255, 0, 1],
+    blue: [0, 0, 255, 1]
   };
+  // Gradient for upward/downward movement TODO Tweak color gradient
```

Comment:
As we discussed the rainbow color scheme is not the best when displaying heights, but this will be easy to change thanks to the way you made it!